src/field_theory/mv_polynomial.lean

```diff
@@ -120,7 +120,7 @@ begin
   exact ⟨1⟩
 end,
 by simp only [indicator, (finset.univ.prod_hom (eval a)).symm, eval_sub,
-    is_ring_hom.map_one (eval a), is_semiring_hom.map_pow (eval a), eval_X, eval_C,
+    is_ring_hom.map_one (eval a), is_monoid_hom.map_pow (eval a), eval_X, eval_C,
```

Comment:
What is the type of `eval a` here? Is it bundled?

Reply:
At this point `eval` is still unbundled.

It's completely straightforward to make `mv_polynomial.eval` a ring homomorphism immediately, _except_ that we lose the notation `p.eval f`, and will have to write `eval f p`. (Perhaps we could do some hackery with an alias?)

Is it okay if we defer that to a second PR? Particularly if you think that writing `eval f p` everywhere will be okay, I can do that easily.

Reply:
I tried making `eval` (and some others) bundled in the branch `mv_polynomial_bundling`, but didn't quite get there; still a few errors.

Perhaps I'll make a draft PR of that and ask for help on the remaining errors after this PR?